cip-0001.md

```diff
@@ -173,4 +171,4 @@ The editors don't pass judgement on CIPs. We merely do the administrative & edit
 
 ## History ##
 
-This document was derived heavily from Bitcoin's [BIP-0001](https://github.com/bitcoin/bips/blob/master/bip-0001.mediawiki), which was derived heavily from Python's PEP-0001. In many places text was simply copied and modified. PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger.  They are not responsible for its use in the Counterparty Improvement Process, and should not be bothered with technical questions specific to Counterparty or the CIP process. Please direct all comments to the [Counterparty forums](https://counterpartytalk.org/c/development) or the [CIP editors](mailto:dweller@devonweller.com).
\ No newline at end of file
+This document was derived heavily from Bitcoin's [BIP-0001](https://github.com/bitcoin/bips/blob/master/bip-0001.mediawiki), which was derived heavily from Python's PEP-0001. In many places text was simply copied and modified. PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger.  They are not responsible for its use in the Counterparty Improvement Process, and should not be bothered with technical questions specific to Counterparty or the CIP process. Please direct all comments to the [Counterparty forums](https://counterpartytalk.org/c/development) or the [CIP editors](mailto:dweller@devonweller.com).
```

Comment:
When you are ready, Robby, let's have you and rubensayshi ACK or noACK this and then I will merge accordingly.